src/groups/mqb/mqbblp/mqbblp_queueengineutil.cpp

```diff
@@ -637,6 +637,17 @@ QueueEngineUtil_AppsDeliveryContext::QueueEngineUtil_AppsDeliveryContext(
 , d_currentMessage(0)
 , d_queue_p(queue)
 , d_timeDelta()
+, d_currentAppView_p(0)
+, d_visitVisitor(
+      bdlf::BindUtil::bindS(allocator,
+                            &QueueEngineUtil_AppsDeliveryContext::visit,
+                            this,
+                            bdlf::PlaceHolders::_1))
+, d_broadcastVisitor(bdlf::BindUtil::bindS(
+      allocator,
+      &QueueEngineUtil_AppsDeliveryContext::visitBroadcast,
+      this,
+      bdlf::PlaceHolders::_1))
```

Comment:
The alternative to this is to made `Visitor` functor and inherit from it locally.
Like here:
https://github.com/bloomberg/blazingmq/pull/481

Reply:
To implement new DeliveryContext interface...